.github/workflows/kbs_build_and_push.yaml

```diff
@@ -0,0 +1,36 @@
+name: Build and Push kbs Image
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Get Commit SHA
+      id: commit_sha
+      run: echo "::set-output name=sha::$(git rev-parse --short ${{ github.sha }})"
```

Comment:
why use the short one?

also I think the new, preferred syntax is: `echo "key=value >> $GITHUB_OUTPUT"`, but I wouldn't put this in the output anyway. it can be an env, or even just inlined in the step below.

Reply:
☝️